### ISSUE TITLE
High volumes of duplicate entries when using Fluentd as the logger

### ISSUE THREAD

**Issue by Jon1760** [dmachard/go-dnscollector#611]
When sending entries to Fluentd, I see hundreds or in some cases thousands of duplicate entries with the same query-ip, query-port, qname, and timestamp-rfc3339ns (reported only to the nearest ms).  This leads to millions of entries with time ranges spanning only a few minutes, vastly exceeding the rate of queries handled by my DNS servers.

I see this both when Fluentd sends the events along to Elasticsearch as well as to a log file. Fluentd isn't doing this to me with any other sources of events, but I've not yet figured out a way to truly rule it out as the culprit.

I do not see this behavior when I have go-dnscollector log directly to a log file.  

My config file:
```
# If turned on, debug messages are printed in the standard output
global:
  trace:
    verbose: true

multiplexer:
  collectors:
    - name: pdns_corp
      dnstap:
        listen-ip: 192.168.6.53
        listen-port: 6000
#    - name: pdns_artist
#      dnstap:
#        listen-ip: 172.16.201.X
#        listen-port: 6000

  loggers:
    # Write DNS logs to a local files in JSON format
    - name: file_corp
      logfile:
        file-path:  "/var/log/dnstap/dnscollector_corp.log"
        mode: json
    - name: file_artist
      logfile:
        file-path:  "/var/log/dnstap/dnscollector_artist.log"
        mode: json
    # Forward to Fluentd in native fluent
    - name: fluentd_corp
      fluentd:
        transport: tcp
        remote-address: 127.0.0.1
        remote-port: 24224
        tag: "dns.corp"
    - name: fluentd_artist
      fluentd:
        transport: tcp
        remote-address: 127.0.0.1
        remote-port: 24224
        tag: "dns.artist"
        

  routes:
    - from: [ pdns_corp ]
      to: [ fluentd_corp ]
#    - from: [pdns_artist ]
#      to: [ file_artist ]
```
where I change the to: value between fluentd_corp and file_corp in the live route to change the behavior.

Given the fact that I get what I expect when go.dnscollector logs directly to a logfile, I do not suspect my dnsdist 1.8.3 servers of sending duplicate events.

Ubuntu 22.04 LTS amd64
Tested with both versions 0.40.2 and 0.41.0-beta2

**Comment by dmachard** [comment]
Thank to report this issue; I can confirm that something it not normal in dnscollector side. 
I need to reproduce before to push a fix for next release.

**Comment by dmachard** [comment]
fixed in beta release v0.41.0beta3

**Comment by dmachard** [comment]
Release v0.41.0 available.
